backend/Users/Users.ts

```diff
@@ -71,3 +64,13 @@ export async function findUser(email, password) {
     return null;
   }
 }
+
+export async function findUserByEmail(email) {
+  const users = await sequelize.models.User.findAll({
+    where: {
+      email: email,
```

Comment:
Hmm emails are case insensitive but databases are not. This api should work regardless of whether the user's casing of their email includes lowercase or uppercase letters. One way to accomplish that is always call .toLowerCase() both when inserting and reading emails from the db. Another way is to tell Postgres (or in our case, sequelize) that it's a case insensitive search. I'd recommend doing that in this case - see how to do case insensitive searches with sequelize

Reply:
I got this to work with ILIKE.  Thank you!

Reply:
Removed ILIKE, used Op.eq instead for exact match.